plutus-core/plutus-core/test/Pretty/Readable.hs

```diff
@@ -31,8 +31,7 @@ test_PrettyReadable =
   where
     folder :: Pretty fun => PlcFolderContents DefaultUni fun -> TestTree
     folder
-        = runTestNestedIn ["plutus-core", "test", "Pretty", "Golden"]
-        . testNested "Readable"
+        = runTestNested ["plutus-core", "test", "Pretty", "Golden", "Readable"]
```

Comment:
`runTestNestedIn` used to be always followed by `testNested`, so I simply removed it and changed all occurrences of the pair to `runTestNested`.